types/lib/callbackRegistry.d.ts

```diff
@@ -0,0 +1,17 @@
+interface ICallback {
```

Comment:
@Kreezag 
I moved all generated declarations in separate dir, this way they are not getting in a way during development
Keeping generated files separately from originals is a good idea generally